src/redux/reducers/usersReducer.js

```diff
@@ -0,0 +1,81 @@
+import { FOLLOW, UNFOLLOW, SET_USERS } from '../actions/actionTypes';
+import { AVATAR_DEFAULT } from '../../utils/url-utils';
+
+const initialState = {
+  users: [
+    {
+      id: '1',
+      photoUrl: { AVATAR_DEFAULT },
+      followed: true,
+      fullName: 'Vasiliy',
+      status: 'vasiliy propionate',
+      location: {
+        city: 'Giran Harbor',
+        country: 'Elmoraden',
+      },
+    },
+    {
+      id: '2',
+      photoUrl: { AVATAR_DEFAULT },
+      followed: true,
+      fullName: 'Gennadiy',
+      status: 'vse dl9 pacanov',
+      location: {
+        city: 'Vorota Diona',
+        country: 'Dion',
+      },
+    },
+    {
+      id: '3',
+      photoUrl: { AVATAR_DEFAULT },
+      followed: true,
+      fullName: 'Tatiyana',
+      status: 'Tanysha podai grabli!!!1',
+      location: {
+        city: 'Minsk',
+        country: 'Belarus',
+      },
+    },
+  ],
+};
+
+const usersReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case FOLLOW:
+      return {
+        ...state,
+        users: state.users.map((user) => {
+          if (user.id === action.userID) {
+            return { ...user, followed: true };
+          }
+          return user;
+        }),
+      };
+
+    case UNFOLLOW:
+      return {
+        ...state,
+        users: state.users.map((user) => {
+          if (user.id === action.userID) {
+            return { ...user, followed: false };
+          }
+          return user;
+        }),
+      };
+
+    case SET_USERS:
+      return {
+        ...state,
+        users: [...state.users, ...action.users],
+      };
+
+    default:
+      return state;
+  }
+};
+
+export const followActionCreator = (userID) => ({ type: FOLLOW, userID });
```

Comment:
Consider either using "in place" exports, or barrel exports.
I suggest moving this exports to the bottom {}.

Reply:
/fxd